### ISSUE TITLE
verify function fail

### ISSUE THREAD

**Issue by wonderwater** [tsndr/cloudflare-worker-jwt#70]
hello, There may be a problem here, it cause verify fail

https://github.com/tsndr/cloudflare-worker-jwt/blob/0cd10e17f7bffe992da34630012382c45ef199e7/src/index.ts#L224-L228

I think `Math.abs` should not be used. Math.abs(...) > 0 almost.

**Comment by wonderwater** [comment]
> What's the error your're getting?

I use version 2.5.0
Basic Example in [[README.md](https://github.com/tsndr/cloudflare-worker-jwt/blob/main/README.md)#examples](https://github.com/tsndr/cloudflare-worker-jwt#examples)
```js
async () => {
    import jwt from '@tsndr/cloudflare-worker-jwt'

    // Creating a token
    const token = await jwt.sign({ name: 'John Doe', email: 'john.doe@gmail.com' }, 'secret')

    // Verifing token
    const isValid = await jwt.verify(token, 'secret')

    // Check for validity
    if (!isValid)
        return

    // Decoding token
    const { payload } = jwt.decode(token)
}

```
I run code above, I got`isValid === false` always

**Comment by oof2win2** [comment]
I am also experiencing this error. I get the `EXPIRED` error thrown here in this case.
```
const JWT_SECRET = "bob"
const apikey = await jwt.sign(
	{
		jti: "123541",
		exp: Math.floor(Date.now() / 1000) + 86400,
		sub: "46",
	},
	JWT_SECRET,
);
const isValid = await jwt.verify(apikey, JWT_SECRET, {
	throwError: true,
});
```

**Comment by tsndr** [comment]
Sorry about that, fixed with [v2.5.1](https://github.com/tsndr/cloudflare-worker-jwt/releases/tag/v2.5.1). Thanks for reporting :)

**Comment by oof2win2** [comment]
Thanks for the quick reply and fix! Also thank you very much for maintaining such a useful library
